ms2/test/src/org/labkey/test/tests/ms2/MS2Test.java

```diff
@@ -1224,4 +1222,33 @@ public Set<String> getFilters()
         }
         return result;
     }
+
+    /**
+     * Issue 52245 - container filter not getting propagated to run group lookup
+     */
+    private void validateContainerFilter()
+    {
+        // create a run group at the project level
+        goToProjectHome(getProjectName());
+        clickAndWait(Locator.linkWithText("Run Groups"));
+        clickButton("Create Run Group");
+        setFormElement(Locator.name("name"), RUN_GROUP3_NAME);

```

Comment:
Should there be a wait after the `clickButton` and before the `setFormElement`?
Like:
`waitForElement(Locator.name("name"));`

Reply:
The page is probably static. Waiting for elements isn't quite as necessary outside of the apps.